src/fft-api/external-actions/fftExternalActionsService.ts

```diff
@@ -0,0 +1,121 @@
+import { CustomLogger } from '../../common';
+import { Logger } from 'tslog';
+import { FftApiClient } from '../common';
+import {
+  ExternalAction,
+  ExternalActionForCreation,
+  ExternalActionForReplacement,
+  ExternalActionLog,
+  ExternalActionLogForCreation,
+  ExternalActionLogs,
+} from '../types';
+import { ResponseError } from 'superagent';
+
+export class FftExternalActionsService {
```

Comment:
we use singular for all other services and directory names
suggest to use `FftExternalActionService` inside `external-action` dir